core/templates/dev/head/pages/exploration_player/supplemental_card_directive.html

```diff
@@ -19,7 +19,10 @@
     </continue-button>
   </div>
 
-  <div>
+  <!--This CSS class is essential for auto-scrolling in Drag and drop interaction to work
+  correctly. More details about the class can be found in extensions/interactions/
```

Comment:
The only "more details" comment I can see there is "This CSS class is essential for auto-scrolling in Drag and Drop Sort interaction to work correctly."

That's not very useful. I think it's better to explain in that file _why_ the CSS class is essential and what will happen if it was removed.